test/e2e/framework/resource_usage_gatherer.go

```diff
@@ -556,3 +557,65 @@ func (g *ContainerResourceGatherer) StopAndSummarize(percentiles []int, constrai
 	}
 	return &summary, nil
 }
+
+// KubemarkResourceUsage is a struct for tracking the resource usage of kubemark.
+type KubemarkResourceUsage struct {
```

Comment:
can this structure become private

`kubemarkResourceUsage`?

Reply:
yes . sorry forget it